mongo/options/clientoptions.go

```diff
@@ -138,6 +138,13 @@ type ClientOptions struct {
 	// release.
 	AuthenticateToAnything *bool
 
+	// Crypt specifies a custom driver.Crypt to be used to encrypt and decrypt documents. The default is no
+	// encryption.
+	//
+	// Deprecated: This option is for internal use only and should not be set. It may be changed or removed in any
+	// release.
```

Comment:
Add a reference to GODRIVER-2149 in the comment to allow future readers to easily discover more context.